FirebaseSessions/Tests/Unit/SessionStartEventTests.swift

```diff
@@ -38,6 +38,24 @@ class SessionStartEventTests: XCTestCase {
     appInfo = MockApplicationInfo()
   }
 
+  var defaultSessionInfo: SessionInfo {
+    return SessionInfo(
+      sessionId: "test_session_id",
+      firstSessionId: "test_first_session_id",
+      dispatchEvents: true,
+      sessionIndex: 0
+    )
+  }
+
+  var thirdSessionInfo: SessionInfo {
+    return SessionInfo(
+      sessionId: "test_session_id",
```

Comment:
For completeness, lets use unique sessionId for each sessionInfo. Easy to ensure that we validate on the right session Ids.